generators/newapp/templates/go.mod.tmpl

```diff
@@ -0,0 +1 @@
+module {{.opts.App.PackagePkg}}
```

Comment:
I'm actually getting errors from this. It looks like `go mod init` is still being called, and because this file already exists, it throws `go.mod already exists`